dev/com.ibm.ws.concurrent.java8/resources/com/ibm/ws/concurrent/resources/CWWKCMessages.nlsprops

```diff
@@ -0,0 +1,87 @@
+CWWKC1002.provider.cardinality.violation.useraction=Remove duplicate context provider elements from the context service configuration.
+
+# Moved from com.ibm.ws.context
+CWWKC1020.baseContextRef.infinite=CWWKC1020E: The attribute baseContextRef of configuration element {0} causes an infinite chain of dependencies.
+CWWKC1020.baseContextRef.infinite.explanation=An infinite chain of dependencies exists because baseContextRef directly or indirectly points back to the same context service instance.
+CWWKC1020.baseContextRef.infinite.useraction=Update the context service configuration to break the infinite chain of dependencies.
+
+CWWKC1100.resource.unavailable=CWWKC1100E: The resource {0} is unavailable.
+CWWKC1100.resource.unavailable.explanation=A resource might become unavailable due to the server shutting down or because of a configuration change, either to itself or another service upon which it depends.
+CWWKC1100.resource.unavailable.useraction=Verify the configuration of the resource and any resources upon which it depends.
+
+CWWKC1110.task.canceled=CWWKC1110I: The task {0}, which was submitted to executor service {1}, is canceled.
+CWWKC1110.task.canceled.explanation=Execution of the task has been canceled.
+CWWKC1110.task.canceled.useraction=No action is required.
+
+CWWKC1101.task.failed=CWWKC1101E: The task {0}, which was submitted to executor service {1}, failed with the following error: {2}.
+CWWKC1101.task.failed.explanation=A task submitted to an executor has failed. Refer to the exception message for the cause.
+CWWKC1101.task.failed.useraction=Correct the cause of the failure and resubmit the task.
+
+CWWKC1102.listener.failed=CWWKC1102E: The managed task listener for task {0}, which was submitted to executor service {1}, failed with the following error: {2}.
+CWWKC1102.listener.failed.explanation=A managed task listener operation has failed. Refer to the exception message for the cause.
+CWWKC1102.listener.failed.useraction=Correct the cause of the failure and decide whether or not to resubmit the task.
+
+CWWKC1103.skip.run.failed=CWWKC1103E: Execution of task {0}, which was submitted to executor service {1}, is skipped because the Trigger.skipRun operation failed with the following error: {2}.
+CWWKC1103.skip.run.failed.explanation=The skipRun operation for the Trigger has failed, causing the current execution attempt for the task to be skipped.
+CWWKC1103.skip.run.failed.useraction=Correct the cause of the failure and decide whether or not to resubmit the task.
+
+CWWKC1111.task.invalid=CWWKC1111E: The task {0} is not valid.
+CWWKC1111.task.invalid.explanation=The task cannot be submitted because it is not valid.
+CWWKC1111.task.invalid.useraction=Update the application to submit a valid task to the ExecutorService.
+
+CWWKC1112.all.tasks.canceled=CWWKC1112I: All tasks are canceled.
+CWWKC1112.all.tasks.canceled.explanation=All of the tasks were canceled before any task could be invoked.
```

Comment:
```suggestion
CWWKC1112.all.tasks.canceled.explanation=All tasks were canceled before any task could be invoked.
```

Reply:
Suggestion: Replace "could be" with "was"